gremlin-driver/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#    or more contributor license agreements.  See the NOTICE file
+#    distributed with this work for additional information
+#    regarding copyright ownership.  The ASF licenses this file
+#    to you under the Apache License, Version 2.0 (the
+#    "License"); you may not use this file except in compliance
+#    with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing,
+#    software distributed under the License is distributed on an
+#    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#    KIND, either express or implied.  See the License for the
+#    specific language governing permissions and limitations
+#    under the License.
+
+version: "3.8"
+
+services:
+
+  gremlin-socket-server:
+    container_name: gremlin-socket-server
+    image: tinkerpop/gremlin-socket-server:${GREMLIN_VERSION}
+    build:
+      context: ../gremlin-tools/gremlin-socket-server/
+      dockerfile: Dockerfile
```

Comment:
I see that you are building the Dockerfile, which is the Socket Server image, again here? I thought it's built every time the Socket Sever module compiles? Shouldn't we just run the prebuilt image instead of building it again?